packages/datagateway-dataview/cypress/support/commands.js

```diff
@@ -54,26 +54,24 @@ export const readSciGatewayToken = () => {
   };
 };
 
-Cypress.Commands.add('login', (username, password) => {
-  return cy.readFile('server/e2e-settings.json').then((settings) => {
-    cy.request('POST', `${settings.apiUrl}/sessions`, {
-      username: username,
-      password: password,
-    }).then((response) => {
-      const jwtHeader = { alg: 'HS256', typ: 'JWT' };
-      const payload = {
-        sessionId: response.body.sessionID,
-        username: 'Robert499',
-      };
-      const jwt = jsrsasign.KJUR.jws.JWS.sign(
-        'HS256',
-        jwtHeader,
-        payload,
-        'shh'
-      );
-
-      window.localStorage.setItem('scigateway:token', jwt);
-    });
+Cypress.Commands.add('login', () => {
+  cy.request({
+    method: 'POST',
+    url: 'https://scigateway-preprod.esc.rl.ac.uk:8181/icat/session',
```

Comment:
It's probably worth adding a TODO to switch this back to the "old" method once we switch over to use the ICAT backend on preprod rather than DB backend, because then that will support anonymous login properly.